README.md

```diff
@@ -0,0 +1,118 @@
+[//]: # "This README.md file is auto-generated, all changes to this file will be lost."
```

Comment:
There's a custom README in this repo, so I think we should add `README.md` to an ignore list in `owlbot.py`.

https://github.com/googleapis/teeny-request/blob/main/owlbot.py#L17